scripts/SHI/scp_SHI_03_HabitatChange_GFW.yml

```diff
@@ -0,0 +1,99 @@
+script: scp_SHI_03_HabitatChange_GFW.R
+name: Measure habitat change
+description: 
+  This script loads the area of habitat of the species and extracts the land cover layers to measure changes on the habitat of the species.
+author: 
+  - name: Maria Isabel Arce-Plata
+    identifier: https://orcid.org/0000-0003-4024-9268 
+license: CC BY 
+external_link: https://github.com/GEO-BON/biab-2.0/tree/main/scripts/SHI 
+inputs:
+  spat_res:
+    label: output spatial resolution
+    description: Spatial resolution (in meters) for the output of the analysis.
+    type: int
+    example: 1000
+  srs:
+    label: spatial reference system
+    description: Reference system for the area of interest. It can be 1) the Spatial Reference System Identifier (SRID), 2) the authority name (e.g. EPSG) with the SRID or 3) the description of the spatial reference system details (e.g. [https://spatialreference.org/]). If just the SRID is given and the software can not find the reference system try options 2 or 3.
+    type: text
+    example: EPSG:3116
+  species:
+    label: species
+    description: Scientific name of the species.
+    type: text[]
+    example: 
+      [
+        "Saguinus oedipus"
+      ]
+  r_area_of_habitat:
+    label: area of habitat
+    description: Raster file with the area of habitat.
+    type: image/tiff;application=geotiff[]
+    example:
+      [
+        "/output/SHI/scp_SHI_02_GetAOH/eccbebe3a9ef4effc2ad36a25380b9ff/Saguinus oedipus/Saguinus oedipus_r_aoh.tif"
+      ]
+  sf_bbox:
+    label: bounding box
+    description: Bounding box for the area of habitat.
+    type: application/geopackage+sqlite3 ; application/geo+json ; application/dbf[]
+    example: 
+      [
+        "/output/SHI/scp_SHI_02_GetAOH/eccbebe3a9ef4effc2ad36a25380b9ff/Saguinus oedipus/Saguinus oedipus_st_bbox.gpkg"
+      ]
+  min_forest:
+    label: min forest
+    description: Minimum tree cover percentage required, based on suitable habitat of the species. Acts as a filter for the Global Forest Watch Data. If not available, use Map of Life Values (e.g. [https://mol.org/species/range/Saguinus_oedipus]) 
+    type: int[]
```

Comment:
this really looks like a mistake in the choice of type : according to the description it's just a single int, not an int[]. (same for max)

Reply:
Same, added that it should be for each species.